client/src/pages/locations/Show.js

```diff
@@ -174,6 +183,30 @@ const LocationShow = ({ pageDispatchers }) => {
                   }}
                   displayType={GEO_LOCATION_DISPLAY_TYPE.FORM_FIELD}
                 />
+
+                <Field
+                  name="attachments"
+                  label={Settings.fields.attachment.shortLabel}
+                  component={FieldHelper.ReadonlyField}
+                  humanValue={
+                    <div
+                      id="attachmentCardList"
+                      style={{
+                        display:
+                          location.attachments.length > 1 ? "flex" : "block"
+                      }}
+                    >
+                      {location.attachments.map((attachment, index) => (
+                        <AttachmentCard
+                          key={index}
+                          attachment={attachment}
+                          index={index}
+                          edit={false}
+                        />
+                      ))}
+                    </div>
+                  }
+                />
```

Comment:
This is the same code as used in `pages/reports/Show/js` so it is a candidate for extraction into a separate component.

Reply:
Let's not split it into a new component for now, but we can evaluate it when the location is finished in the attachment, but I don't think we need to separate this part as a component again. Because although it is not that big, many similar structures exist in forms, so my opinion is not compulsory for now.